src/e2eFlutterTest/java/io/appium/java_client/android/CommandTest.java

```diff
@@ -115,4 +119,26 @@ void testDragAndDropCommand() {
         assertEquals(driver.findElement(AppiumBy.flutterText("The box is dropped")).getText(), "The box is dropped");
 
     }
+
+    @Test
+    void testCameraMocking() throws IOException {
+        driver.findElement(BaseFlutterTest.LOGIN_BUTTON).click();
+        openScreen("Image Picker");
+
+        final String successQr = driver.injectMockImage(
```

Comment:
how injected images are managed on the server side? Is their amount limited? is there any limitation for the max amount/size of these images?

Reply:
Currently we don't have any checks to limit the num of images / size of the image. But It should be a simple tweak on the server side.

Reply:
I have created a issue in flutter-driver to add check on limit and file size. https://github.com/AppiumTestDistribution/appium-flutter-integration-driver/issues/77